### DISCUSSION TITLE
Weird 0% showing in the tool tip

### DISCUSSION THREAD

**Discussion by monsterCoder1012** [Live-Charts/LiveCharts2#1358] [Q&A]
I am using the .Net Framework version. I am trying to track down why 0.00% is showing. I had to make a change to make a slight change to Live Charts code to handle me wanting to send the Legends I wanted to show. Since I have noticed this weird percent showing. Where in the code does it create this string in the ToolTip?

![image](https://github.com/beto-rodriguez/LiveCharts2/assets/5554132/fc1cd3d0-f200-4d35-8e6b-79d9e381cacf)

**Comment by monsterCoder1012** [comment]
@beto-rodriguez Ok I found the section that adds the 0.00%. What is the purpose of this section?

```
<TextBlock Grid.Column="3" Text="{Binding ChartPoint.Participation, StringFormat={}{0:P}}" 
       VerticalAlignment="Center" Margin="5 0 0 0"
       Visibility="{Binding DataContext.Data, RelativeSource={RelativeSource  FindAncestor, 
                                                            AncestorType={x:Type StackPanel}}, 
                            Converter={StaticResource ParticipationVisibilityConverter}}"/>
```

**Reply by beto-rodriguez** [reply]
Hi, can you please attach an image?
